lib/easy_meli/error_parser.rb

```diff
@@ -29,8 +31,8 @@ def self.status_error_class(response)
 
   private
 
-  def self.error_message_from_body(response)
-    response['message'].to_s.empty? ? response['error'] : response['message']
+  def self.find_in_error_list(response_field)
+    ERROR_LIST.find { |key, _| response_field&.include?(key) }&.last
```

Comment:
Podemos hacer esto? Así continuamos con la validación que teníamos antes en `response['message']`

```suggestion
    return if response_field.to_s.empty?
    
    ERROR_LIST.find { |key, _| response_field.include?(key) }&.last
```